assets/icons/preview.html

```diff
@@ -0,0 +1,687 @@
+<html>
```

Comment:
Can you show me where we used this file? I did a global search but didn't find... Might miss something.

Reply:
We not really using this in any of our code, 
I have a hard time to preview all SVG icons, so I generated this preview.html using a Python script, and by looking at this preview.html, I can easily tell what does each SVG looks like.

I can also add the Python Script I used, if needed.

Reply:
this file has been removed.